components/event-bus/internal/trace/tracer.go

```diff
@@ -70,39 +81,36 @@ func (zk *OpenTracer) Start() {
 	}
 }
 
+// Started returns a boolean value indicating id the OpenTracer is started or not.
 func (zk *OpenTracer) Started() bool {
```

Comment:
If it not out of the scope, I would rename the function to `isStarted()`

Reply:
**isStarted()** is incorrect, it has to be exported for external usage like this **IsStarted()**.

Reply:
👍